contracts/DecodeBlock.sol

```diff
@@ -158,7 +356,22 @@ library DecodeBlock {
         array = item.toAddressArray();
     }
 
-    function _decodeNextValidatorPowers(RLPReader.RLPItem memory item) private pure returns (uint64[] memory array) {
-        array = item.toUint64Array();
+    function _decodeNextValidatorPowers(RLPReader.RLPItem memory item) private pure returns (uint256[] memory array) {
+        array = item.toUintArray();
+        return toOnChainPowers(array);
+    }
+
+    function toOnChainPowers(uint256[] memory array) internal pure returns(uint256[] memory){
+        for (uint256 i = 0; i < array.length; i++) {
+            array[i] = array[i] * STAKE_UINT;
+        }
+        return array;
+    }
+
+    function toOffChainPowers(uint256[] memory array) internal pure returns(uint256[] memory){
```

Comment:
I feel toOffChainPowers/toOnChainPowers are quite confusing (as toOffChainPowers essentially provides powers for Web3Q on-chain).

I am wondering whether we could use toTendermintPowers(), which is easier to understand.